packages/com.circleci.v2/Config.pkl

```diff
@@ -218,6 +220,7 @@ typealias ResourceClass =
   |"gpu.nvidia.medium"
   |"gpu.nvidia.large"
   |"windows.gpu.nvidia.medium"
+  |String
```

Comment:
Added `String` here to avoid needing to update the package to match CircleCI changes in the future.